pyracf/scripts/setup_precheck.py

```diff
@@ -0,0 +1,34 @@
+from pyracf import ResourceAdmin, SecurityRequestError
+
+
+def setup_precheck():
+    resource_admin = ResourceAdmin()
+    try:
+        access = resource_admin.get_my_access("IRR.IRRSMO00.PRECHECK", "XFACILIT")
+    except SecurityRequestError:
+        traits_precheck = {"base:universal_access": "None"}
+        result = resource_admin.add(
+            "IRR.IRRSMO00.PRECHECK", "XFACILIT", traits=traits_precheck
+        )
+        print(
+            "IRR.IRRSMO00.PRECHECK is now defined with a `Universal Access` of None."
+            + "\nContact your security administrator for READ access before using pyRACF."
+            + "\nOther users of pyRACF will also need to have at least read access."
+            + "\nYou may also need to REFRESH the `XFACILIT` class."
+            + "\nReview our documentation at https://ambitus.github.io/pyracf/ as well!"
+        )
```

Comment:
`\n` should go at the end of the previous line instead of at the beginning of the current line.

```python
        print(
            "IRR.IRRSMO00.PRECHECK is now defined with a `Universal Access` of None.\n"
            + "Contact your security administrator for READ access before using pyRACF.\n"
            + "Other users of pyRACF will also need to have at least read access.\n"
            + "You may also need to REFRESH the `XFACILIT` class.\n"
            + "Review our documentation at https://ambitus.github.io/pyracf/ as well!"
        )
```

Reply:
sure, fixed.